### PR TITLE
Added topbanner

### PR DESCRIPTION
**Pull request by Winkeltje84**
screen shot 2017-03-28 at 11 47 39

looks like this now
screen shot 2017-03-28 at 14 03 13

### CODE REVIEW

File: app/views/topbanner.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
+
+<section>
+  <div class="container">
+    <div class="jumbotron" align="middle">
+      <img src="http://res.cloudinary.com/dqaserzum/image/upload/v1490689989/Screen_Shot_2017-03-28_at_10.29.07_pv9ybs.png" height="125" alt="'Topp banner Taste of Code" />
```
Comment (anikaSchwing) [inline app/views/topbanner.html.erb]: "'Topp banner Taste of Code" there is a small typo here, maybe change it to "Top banner Taste of Code"

Comment (anikaSchwing) [inline app/views/topbanner.html.erb]: "'Topp banner Taste of Code" there is a small typo here, maybe change it to "Top banner Taste of Code"

File: app/views/topbanner.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      <p class="lead" style="padding:10px">Beginner Coding Workshop</p>
```
Comment (anikaSchwing) [inline app/views/topbanner.html.erb]: In the wireframe it also says 'Taste of Code' above 'Beginner Coding Workshops' please add that too :)

**Comment by anikaSchwing** [comment]
Also I almost forgot: is this a partial? Because in that case I think the filename needs an _ at the start

File: app/views/topbanner.html.erb (hunk 1 above)
Comment (arnoFleming) [inline app/views/topbanner.html.erb]: if you want to render images that you added yourself, please add them to your `app/assets/images` and use `image_tag`

Comment (Winkeltje84) [inline app/views/topbanner.html.erb]: fixed it!

File: app/views/topbanner.html.erb (hunk 2 above)
Comment (Winkeltje84) [inline app/views/topbanner.html.erb]: added it, but title is already in Logo, so we would only want to show it if it is not in the logo I think

File: app/views/topbanner.html.erb (hunk 1 above)
Comment (Winkeltje84) [inline app/views/topbanner.html.erb]: added image to image folder now and reffered to it in html

File: app/views/shared/_topbanner.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+</head>
+
```
Comment (miriamtocino) [inline app/views/shared/_topbanner.html.erb]: In a partial you don't need the `<head>` tag. That's only in your layout.

Please remove it from here 😄

Comment (Winkeltje84) [inline app/views/shared/_topbanner.html.erb]: yes, did that to make the page visible by just looking at that single html file. Will delete it!

**Pull request merged**